extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -192,70 +196,72 @@ export class AmpLiveList extends AMP.BaseElement {
   buildCallback() {
     this.viewport_ = this.getViewport();
 
-    LiveListManager.forDoc(this.element).then(manager => {
-      this.manager_ = manager;
-      this.manager_.register(this.liveListId_, this);
-    });
+    return this.isExperimentEnabled_().then(enrolled => {
+      LiveListManager.forDoc(this.element, enrolled).then(manager => {
```

Comment:
Will `enrolled` propagate to the `LiveListManager` constructor? https://github.com/ampproject/amphtml/blob/f49ba4edc8d4d280b191793ddd4ecca1b2f0e028/extensions/amp-live-list/0.1/live-list-manager.js#L123-L128

Maybe we should do this check in `LiveListManager` on the first `register()` call instead. You can get an element from `liveList.element` in that function.

Reply:
Can refactor this way, will be a little bit.

Reply:
Moved over, mind taking another look?